tests/stdlib/metaprogramming/mgenast.nim

```diff
@@ -31,6 +31,7 @@ macro bindme6UseExpose*(): untyped =
   genAst:
     var tst = "sometext"
     var ss = newStringStream("anothertext")
+    prepareMutation(tst)
```

Comment:
Promise me that if we do something with copy on write it has to be named `prepareMootation`. 🐄